Formula/librealsense@1.rb

```diff
@@ -0,0 +1,31 @@
+class LibrealsenseAT1 < Formula
+  desc "Intel RealSense F200, SR300, R200, LR200 and ZR300 capture"
+  homepage "https://github.com/IntelRealSense/librealsense/tree/legacy"
+  url "https://github.com/IntelRealSense/librealsense/archive/v1.12.1.tar.gz"
+  sha256 "62fb4afac289ad7e25c81b6be584ee275f3d4d3742468dc7d80222ee2e4671bd"
+  head "https://github.com/IntelRealSense/librealsense.git", :branch => "legacy"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "glfw"
+  depends_on "libusb"
+
+  def install
+    system "cmake", "-DBUILD_EXAMPLES=true", *std_cmake_args
```

Comment:
does `=ON` work here? That is usually what we have

Reply:
Yep, just tested and updated.